extra-resources/mods/Mythos/chinese.cardset

```diff
@@ -239,10 +226,9 @@ card('SHINJE') {
     }
 }
 
-// Info: http://en.wikipedia.org/wiki/Guanyin
-// Image: http://upload.wikimedia.org/wikipedia/commons/8/89/Daienin_Kannon.JPG
-// License: CC BY-SA 3.0
-// Author: ':...---...SOS' | http://commons.wikimedia.org/w/index.php?title=User:...---...SOS&action=edit&redlink=1
+// https://en.wikipedia.org/wiki/Guanyin
+// https://commons.wikimedia.org/wiki/File:Daienin_Kannon.JPG
+// Attribution: By No machine readable author provided. ...---...SOS assumed (based on copyright claims). [<a href="http://www.gnu.org/copyleft/fdl.html">GFDL</a> or <a href="http://creativecommons.org/licenses/by-sa/3.0">CC BY-SA 3.0</a>], <a href="https://commons.wikimedia.org/wiki/File%3ADaienin_Kannon.JPG">via Wikimedia Commons</a>
```

Comment:
this doesn't look correct... a bit too many licenses and HTML tags listed here. Please clean up (and pick the CC BY-SA license)